sphinx/developers/tool-developer-guide.rst

```diff
@@ -102,7 +102,7 @@ but current thinking is to have different categories:
     * The next level would include class inheritance and slots, and
       basic enums
     * The level above this would include slot inheritance, and slot
-      overrides with slot_usage
+      overrides with `slot_usage`
```

Comment:
we could go one better and have links to the URIs for each concept... but this is great for now